### DISCUSSION TITLE
调查一下大家都用什么GPU？

### DISCUSSION THREAD

**Discussion by maxvint** [filecoin-project/community-china#51] [矿机配置]
来个小调查，大家都用什么GPU挖矿？在下面跟帖回复：
- 用什么GPU (NVIDIA、AMD、其他)？
- winning-post的时间？
- PC2的时间？
- C2的时间

例如：
- GeForce RTX 3090 (NVIDIA GPU通过`nvidia-smi -L`查看)
- 4s
- 15m
- 12m

**Comment by kaptinlin** [comment]
优化后的数据还是官方原版的数据？

**Reply by maxvint** [reply]
优化后的数据呗，现在很少人用官方原版程序吧...

**Reply by 0xhhhhh** [reply]
> 优化后的数据呗，现在很少人用官方原版程序吧...

表示我在用- -。有开源版本的优化后的吗？

**Reply by KaixuanZhao** [reply]
IPFS主网最大雅典娜矿池，提供一站式采购托管运营，fil存量数据迁移出海等服务，全网算力领先1400PB，需要B端1PB以上，C端50T，100T，200T（可扩容）和FIL存量数据迁移出海服务方案的滴滴我，欢迎加入社群Wechet：kkmmjjnn00

**Comment by ldjdsc** [comment]
GeForce RTX 2080Ti
6-8s
25m
50m

**Reply by ifanheng** [reply]
C2居然比P2慢这么多吗

**Comment by linfeicui** [comment]
GeForce RTX 3080

**Comment by KaixuanZhao** [comment]
RRMine，雅典娜矿池，全网算力领先1000PB，需要B端C端方案的滴滴我，交流学习，欢迎加入社群Wechet：kkmmjjnn00

**Comment by GZpowerleader** [comment]
宝德矿机软硬一体方案滴滴我18589289061

**Comment by Chenhongwe** [comment]
Fil算力填充，240/T，包半年运维，送七牛商业存储软件一年，坐标杭州，欢迎骚扰 phone：15255579473（同V）

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308
